scripts/build/build.sh

```diff
@@ -30,7 +30,8 @@ pushd "$MAVEN_DIR"
 echo "-------------------------------------------------------------------------------"
 echo "Building Project"
 echo "-------------------------------------------------------------------------------"
-echo "Running Maven in quiet mode, this may take several minutes."
+echo "Running Maven in quiet mode."
+echo "For the first build, this should take up to 20 minutes. Subsequent builds should take closer to 4 minutes."
```

Comment:
The first build can take up to an hour on the EC2 we deploy with the `sleeper environment` CLI. Subsequent builds take more like 10 minutes. Maybe we should give a range of times?

It might also be worth noting that you can disable the Rust build by passing `-DskipRust` as an argument.